### ISSUE TITLE
Initial bundle columns immutable

### ISSUE THREAD

**Issue by leondz** [codalab/codalab-competitions#2755]
The leaderboard column name set in the initial YAML upload can't be overridden.

Sample competition here: 

https://competitions.codalab.org/competitions/22998#results

If the initial YAML has, for example:

```
leaderboard:
    leaderboards:
        Results: &RESULTS
            label: Results
            rank: 1
    columns:
        score:
            leaderboard: *RESULTS
            label: F1 Macro
            numeric_format: 3
            rank: 2
```

Then the "score" field cannot be changed. The leaderboard column can be changed through the "edit" page, but the change is silently rejected & ignored, which is tricky to troubleshoot.

-- Thanks for your continued work on CodaLab!

**Comment by Didayolo** [comment]
Hi,

Thank you very much for reporting your issue!

Indeed, this issue has been mentionned here: #2686 

Currently, the only work-around I found is to re-upload the bundle, unfortunately.

**Comment by Didayolo** [comment]
I close this issue as it is merged with #2686
